### PR TITLE
test(orchestrator): add flaky liveness replay fixture

### LINKED ISSUE #1806: test(stability): add flaky liveness fixture replay harness
## Description
Implement an LLM-friendly, independently shippable backlog item for **add flaky liveness fixture replay harness**. This issue should stay narrow enough for one agent/PR while producing durable behavior, tests, and documentation where appropriate.

## Affected locations
- Dispatcher, liveness, worker lifecycle, and state handling
- Test harnesses and failure-path fixtures
- Runtime cleanup and observability paths

## Impact
Without this work, future agents and operators will keep rediscovering the same gap, creating avoidable coordination cost, reliability risk, or ambiguous handoffs.

## Recommendation
Implement the scoped behavior with deterministic tests, concise docs, and structured output that PM/liveness tooling can consume.

## Goals
Deliver add flaky liveness fixture replay harness as a discrete capability or hardening change.
Make the behavior deterministic enough for autonomous workers to verify.
Keep the implementation small enough for a one-issue worker and one PR.
Capture operator/user-facing behavior in docs or help text when applicable.

## Expected outcomes
add flaky liveness fixture replay harness is available, documented, and test-backed.
Future PM/worker handoffs can point at this issue without needing extra interpretation.
Failure modes are explicit and produce actionable messages instead of silent drift.
The change does not broaden into adjacent backlog work or unrelated refactors.

## Suggested implementation slices
Inventory the current code/docs paths that own this behavior.
Add or update the narrow model/config/schema needed for the feature or guardrail.
Implement the smallest runtime/docs change that satisfies the acceptance criteria.
Add focused fixtures/tests for success, failure, and edge cases.
Update docs/help output and handoff notes so later agents can use it.

## Acceptance criteria
- [ ] There is a dedicated implementation or documentation change for add flaky liveness fixture replay harness.
- [ ] At least one automated test, fixture, or deterministic verifier covers the intended behavior.
- [ ] At least one negative/edge case is covered so the failure mode is explicit.
- [ ] Docs/help text or inline operator guidance explains how to use or interpret the behavior.
- [ ] The PR description links this issue and summarizes verification evidence.
- [ ] No unrelated production behavior is changed outside the stated scope.

## LLM handoff notes
- Treat this as a one-issue/one-PR unit.
- Start by searching for existing implementations and tests before adding new surfaces.
- Prefer structured outputs and fixtures over prose-only behavior.
- If this overlaps an existing open issue, comment with evidence and stop for PM deduplication instead of creating an unscoped implementation.
- Do not perform broad refactors or change unrelated worker/PM behavior while implementing this ticket.

## Source
Created by the 2026-07-11 fourth strategic backlog expansion run requesting 100 issues. Topic: **Stability**.

### PR DESCRIPTION
**Pull request by djm204**
## Summary
- add a deterministic flaky liveness replay fixture for issue-worker backpressure/routing snapshots
- replay the fixture through `evaluateIssueBackpressure` and degraded-mode worker routing, including recovery and unrelated-dependency edge cases
- document how to add and run flaky liveness replay cases

## Verification
- `npm test --workspace=@franken/orchestrator -- tests/unit/issues/issue-runner.test.ts`
- `npm run build --workspace=@franken/types`
- `npm run build --workspace=@franken/observer`
- `npm run build --workspace=@franken/brain`
- `npm run build --workspace=@franken/critique`
- `npm run build --workspace=@franken/governor`
- `npm run build --workspace=@franken/planner`
- `npm run typecheck --workspace=@franken/orchestrator`
- `npm run build --workspace=@franken/orchestrator`
- `npm run lint --workspace=@franken/orchestrator` (passes with existing warnings)

Closes #1806

### CODE REVIEW

**Comment by djm204** [comment]
@codex review

File: packages/franken-orchestrator/tests/unit/issues/issue-runner.test.ts [hunk 1]
```diff
@@ -588,6 +618,45 @@ describe('IssueRunner', () => {
...
+      const fixture = readFlakyLivenessReplayFixture();
+      const replayCases = [...fixture.snapshots, ...fixture.edgeCases];
+
+      expect(fixture.description).toContain('Flaky liveness replay fixture');
+      expect(fixture.issue.number).toBe(1806);
+      expect(replayCases.map(testCase => testCase.name)).toEqual([
+        'process-capacity-spike-defers-fresh-worker',
+        'github-flake-resumes-checkpointed-worker',
+        'recovered-liveness-starts-fresh-worker',
+        'unconfigured-unrelated-dependency-does-not-pause-liveness',
```
[the remark this answers is not in the document]
Comment (djm204) [reply packages/franken-orchestrator/tests/unit/issues/issue-runner.test.ts]: Fixed in 49a69aa0: the baseline-case assertion now uses expect.arrayContaining(...), so future fixture snapshots can be added without editing the test while the required issue #1806 baseline cases remain guarded.

File: packages/franken-orchestrator/tests/unit/issues/issue-runner.test.ts [hunk 2]
```diff
@@ -588,6 +618,45 @@ describe('IssueRunner', () => {
...
+            cumulativeTokens: 0,
+            budgetTokens: 1_000_000,
+            providerBudgetTokensRemaining: 1_000_000,
+          },
+        );
+        const route = routeIssueWorkerForDegradedMode({
+          issue: makeIssue(fixture.issue),
+          checkpointHasIssueProgress: testCase.checkpointHasIssueProgress,
+          graphHasCheckpointProgress: testCase.graphHasCheckpointProgress,
+          backpressureDecision: decision,
```
[the remark this answers is not in the document]
Comment (djm204) [reply packages/franken-orchestrator/tests/unit/issues/issue-runner.test.ts:660]: Fixed in 49a69aa0: checkpointed replay snapshots can now carry the prior stopRemainingReason into routeIssueWorkerForDegradedMode, matching the production path where checkpointed degraded routes are derived from an existing stop reason. The dependency-breaker fixture case models that prior stop reason explicitly.

**Comment by djm204** [comment]
@codex review
